_articles/open-firmware-systems.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: article
+title: System76 Open Firmware Models
+description: >
+  List of our systems that run our Open Firmware and Open EC
+keywords:
+  - Laptop
+image: http://support.system76.com/images/system76.png
+hidden: false
+section: hardware-drivers
+
+---
+
+### System76 Open Firmware 
+
+When we say Open Firmware we are using the following technologies:
+
+- coreboot
+- EDK2
+- System76 Firmware Apps
+
+Here is a [link](https://github.com/system76/firmware-open) to the source code for our Open Firmware.
+
+### System76 Open EC
+
+When we say Open EC we are talking about the <b>E</b>mbedded <b>C</b>ontroller firmware on our system which handles components on the system (laptop):
+
+- Keyboard
+- Touchpad 
+- Power Button.
+
+Here is a [link](https://github.com/system76/ec) to the source code for our Open EC.
+
+<table>
+<tbody>
+<tr>
+<td>System </td>
+<td>Open Firmware</td>
+<td>Open EC</td>
+</tr>
+<tr>
+<td>galp5</td>
+<td>Yes</td>
+<td>Yes</td>
+</tr>
+<tr>
+<td>galp4</td>
+<td>Yes</td>
+<td>No</td>
+</tr>
+<tr>
+<td>lemp10</td>
+<td>Yes</td>
+<td>Yes</td>
+</tr>
+<tr>
+<td>lemp9</td>
```

Comment:
What order are these being sorted by? I would normally prefer alphabetical order

Reply:
I don't believe there is a real order though I did put the newer versions at the top. I'll correct that in the next commit.

Reply:
New commit has the table list updated and in markdown instead of HTML.